Realm/RLMSchema.mm

```diff
@@ -95,6 +95,9 @@ @implementation RLMSchema {
         s_sharedSchema.objectSchemaByName[schema.className] = schema;
     }
 
+    if ([cls shouldIncludeInDefaultSchema] && schema.isSwiftClass && schema.properties.count == 0) {
```

Comment:
This isn't the right place for this check. It should go with the existing validation logic in RLMObjectSchema.mm.